projects/opendr_ws/src/ros_bridge/src/opendr_bridge/bridge.py

```diff
@@ -85,6 +89,34 @@ def to_ros_pose(self, pose):
             keypoints.detections.append(keypoint)
         return keypoints
 
+    def to_ros_3Dpose(self, pose):
```

Comment:
If you're adding methods to the rosbridge, they should be documented in the corresponding [documentation](https://github.com/tasostefas/opendr_internal/blob/master/docs/reference/rosbridge.md)